.github/workflows/CD.yml

```
@@ -0,0 +1,37 @@
+name: CD
+
+on:
+    push:
+      tags:
+        - '**'
+
+jobs:
+
+  conda:
+    name: Build with conda and upload
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+            fetch-tags: true
+            fetch-depth: 0 
+
+      - name: Setup conda environment
+        uses: conda-incubator/setup-miniconda@11b562958363ec5770fef326fe8ef0366f8cbf8a
+        with:
+          miniconda-version: "latest"
+          python-version: ${{ vars.PY_VERSION }}
+          environment-file: .conda/env_build.yml
+          auto-update-conda: false
+          auto-activate-base: false
+          show-channel-urls: true
+
+      - name: Build and upload the conda package
+        uses: uibcdf/action-build-and-upload-conda-packages@c6e7a90ad5e599d6cde76e130db4ee52ad733ecf
+        with:
+          meta_yaml_dir: conda
```

This still needs to be modified I think:

```suggestion
          meta_yaml_dir: .conda
```

---

Oh shoot I thought I modified them all! 😮

---

Done!